docs/tutorials/motion/plan-motion-with-arm-gripper.md

```diff
@@ -0,0 +1,431 @@
+Feel free to change these dimensions, including thickness (the Z coordinate in the above code samples), to match your environment more closely.
+Additional obstacles can also be *appended* as desired.
+
+{{< alert title="Tip" color="note" >}}
+Within the App, the **Frame System** tab in the **Config** section of your robot gives you the ability to experiment with various geometry representations with better visual feedback.
+{{< /alert >}}
+
+<!-- TODO : Add a picture example of a geometry specified through the Frame System tab -->
+
+## Commanding an Arm to Move with the Motion Service
+
+Commanding motion with the Motion service has a more general feel than previous examples that were commanding motion for individual components.
+The Motion's `Move` method is what you will use when executing more general robotic motion.
+You can designate specific components to plan motion for by passing in the resource name (note the use of the arm resource in the code samples below).
+The `worldState` we constructed earlier is also passed in so that the Motion service can consider additional information when planning.
+
+The sample pose given below can be adjusted to fit your specific circumstances.
+Remember that X, Y, and Z coordinates are specified in millimeters.
+
+Again, a note:
+
+{{< alert title="Caution" color="caution" >}}
+Executing code presented after this point *will* induce motion in a connected robotic arm!
+{{< /alert >}}
+<br><br>
+{{< tabs >}}
+{{% tab name="Python" %}}
+
+<!-- TODO: Add Python -->
+```python {class="line-numbers linkable-line-numbers"}
+```
+
+{{% /tab %}}
+{{% tab name="Go" %}}
+Because of experimental API changes that are ongoing, we must pass in a `slam` service resource name when using the Motion service to Move.
+Add `"go.viam.com/rdk/services/slam"` to your import list to provide access to the `slam` service.
+This tutorial will not cover any other SLAM content.
+
+```go {class="line-numbers linkable-line-numbers"}
+// Generate a sample "start" pose to demonstrate motion
+testStartPose := spatialmath.NewPose(
+  r3.Vector{X: 510.0, Y: 0.0, Z: 526.0},
+  &spatialmath.OrientationVectorDegrees{OX: 0.7071, OY: 0.0, OZ: -0.7071, Theta: 0.0},
+)
+testStartPoseInFrame := referenceframe.NewPoseInFrame(referenceframe.World, testStartPose)
+_, err = motionService.Move(context.Background(), myArmResource, testStartPoseInFrame, worldState, nil, slam.Named(""), nil)
+if err != nil {
+  logger.Fatal(err)
+}
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+<!-- TODO : In the future, we should add some specific information on the importance of the frame chosen as the point of reference for PoseInFrame variables -->
+<!-- ## Managing Pose References -->
+
+## Commanding Other Components to Move with the Motion Service
+
+This section will require you to add a new component to your robot.
+One device that is very commonly attached to the end of a robot arm is a **gripper**.
```

Comment:
Ah, good question. This may be the time to start doing so. At a minimum, users should be comfortable adding a fake gripper to their robot.

This is truly where Motion Planning starts to become useful, and definitely needs to be part of this tutorial.

Reply:
Hmm. Is there a specific person I should follow up with?